Source/KakapoServer.swift

```diff
@@ -9,9 +9,34 @@
 import Foundation
 
 /**
- A server that conforms to NSURLProtocol in order to intercept outgoing network communication
+ A server that conforms to NSURLProtocol in order to intercept outgoing network communication.
+ You shouldn't use this class directly but register a `Router` instead. Since frameworks like **AFNetworking** and **Alamofire** require manual registration of the `NSURLProtocol` classes you will need to register this class when needed.
+
+ ### Examples
+ 
+ 1- Configure `NSURLSessionConfiguration` by adding `KakapoServer` to `protocolClasses`:
+ 
+ ```
+ let configuration = NSURLSessionConfiguration.defaultSessionConfiguration()
+ configuration.protocolClasses = [KakapoServer.self]
+ // NOTE: better to just add if is not nil
+```
+ 
+ 2- Setup the URL Session Manager
+ 
+ #### AFNetworking
+ 
+ ```
+ let manager = AFURLSessionManager(sessionConfiguration: configuration)
+ ```
+ 
+ #### Alamofire
+ 
+ ```
+ let manager = Manager(configuration: configuration)
+ ```
```

Comment:
You still need to register the Router here right? Also I don't think this should be part of this documentation but probably of the README.